src/test/scala/nl.knaw.dans.easy.deposit/servlets/HappyRoutesSpec.scala

```diff
@@ -50,9 +50,10 @@ class HappyRoutesSpec extends TestSupportFixture with ServletFixture with Scalat
       uri = "/auth/login",
       params = Seq(("login", "foo"), ("password", "bar"))
     ) {
-      status shouldBe OK_200
-      body shouldBe "signed in"
+      status shouldBe NO_CONTENT_204
+      body shouldBe ""
       header("Set-Cookie") should startWith("scentry.auth.default.user=") // details in TypicalSessionSpec
+      header("Set-Cookie") shouldNot startWith("scentry.auth.default.user=;")
```

Comment:
what's the difference here with the line above? just the `;`? can you explain when THIS line would be the case? (in other words, when would I get `header("Set-Cookie") should startWith("scentry.auth.default.user=;")`)